recipes/bagit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+# https://github.com/LibraryOfCongress/bagit-python/issues/70
+{% set name = "bagit" %}
+{% set version = "1.5.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/LibraryOfCongress/bagit-python.git
+  git_rev: 613a82849d0dd43761e005606cd4c7ba3cd44188
```

Comment:
They are using `setuptools-scm` to version... which was new to me. It needs the actual git repo to run... unless we want to patch those portions out.

```
LookupError: setuptools-scm was unable to detect version for '/data/miniconda3/conda-bld/work/bagit-python-613a82849d0dd43761e005606cd4c7ba3cd44188'.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.
```